.travis.yml

```diff
@@ -1,6 +1,6 @@
 language: node_js
 node_js:
-  - "stable"
+  - 10.15.1
```

Comment:
Wont this specific version fail travis again if there is a newer node version

Reply:
No, This is the stable version that we have been using before the upgrade of Node version. I am very certain travis won't fail again.